stories/Components/TableStoriesDocs/TableDocs.mdx

```diff
@@ -1,42 +1,59 @@
 `import { Table } from "@bigbinary/neetoui";`
 
-We use Ant Design's `Table` component under the hood. For extra customization, refer [AntD Table](https://ant.design/components/table/#API).
+```jsx
+const columnData = [
+  {
+    title: "ID",
+    dataIndex: "id",
+    key: "id",
+    width: 75,
+    sorter: (a, b) => a.id - b.id,
+  },
+  // Other column defenitions
+  ]
+
+const rowData = [
+  {
+    id: "1",
```

Comment:
Is `id` supposed to be a `string` or a `number`? If it's a string, then would the sorting condition that you've written above generate the desired outcome?

Reply:
Technically it should work because of the type-coercion. But this can cause confusion, So will be changing it.